app/models/bookmark.js

```diff
@@ -1,23 +1,21 @@
-import DS from 'ember-data';
+import DS from 'ember-data'; // eslint-disable-line
 import { computed } from '@ember/object';
 
-const {
-  PromiseObject,
-  Model,
-  attr,
-  belongsTo,
-} = DS;
+// This throws https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/use-ember-data-rfc-395-imports.md.
+// Fixing this elsewhere in the code was easy but I'm not sure where to get PromiseObject from.
+// If I import it off of @ember-data/model, it's undefined.
+const { PromiseObject, Model, attr, belongsTo } = DS; // eslint-disable-line
```

Comment:
@allthesignals  would appreciate some help on this as well. I was able to easily resolve this linting rule elsewhere but something funky is going on with `PromiseObject`.